common/requirements-common.txt

```diff
@@ -1 +0,0 @@
-../requirements-common.txt
```

Comment:
@mgorsk1 it seems the depenbot which upgrades version to solve CVE won't work any more with the new approach?